kne_cli/main.go

```diff
@@ -45,7 +45,8 @@ func main() {
 		}
 	}
 	flags.Import()
-	err := cmd.ExecuteContext(context.Background())
+	root := cmd.New()
+	err := root.ExecuteContext(context.Background())
```

Comment:
You could just say `cmd.New().ExecuteContext(...)`

Reply:
done